code/modules/client/loadout/loadout_accessories.dm

```diff
@@ -105,17 +106,21 @@
 //necklaces and shiz
 
 /datum/gear/accessory/petcollar
+	slot = ITEM_SLOT_NECK
 	display_name = "pet collar"
 	path = /obj/item/clothing/neck/petcollar
 	cost = 20000
 
 /datum/gear/accessory/necklace
+	slot = ITEM_SLOT_NECK
 	display_name = "dope necklace"
 	path = /obj/item/clothing/neck/necklace/dope
 	cost = 25000
 
 /datum/gear/accessory/oldnecklace
+	slot = ITEM_SLOT_NECK
 	display_name = "necklace, gold"
+	description = "All about that bling" //Just so it isn't completely obvious this is the exact same item as above
```

Comment:
why are there two of the same exact thing though

Reply:
I asked the same thing in chat. The answer I got is that allegedly one of them used to be a different icon which got removed for some reason. 

I can't simply yoink one of them from the shop because that will cost a good handful of players access to an item they paid 20,000 coins for.

Reply:
Hmm okay, we could make another PR and some sql to fix that but good for now